### ISSUE TITLE
Fix XHTML markup in doc/core

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5018]
|<img alt="techtonik's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| techtonik reported|
|-|-|
|Trac ID|trac#5018|
|Type|enhancement|
|Created|2011-04-03 18:06:03Z|

This patch fixes XHTML markup errors for `doc/core/development/` For example, XHTML doesn't allow <pre> nested in <p>. I also wrapped some long lines during review.

`doc\core\howto\pb-usage.xhtml` contained `<span class="footnote">` with `<p>` inside. I removed `<p>`, but haven't seen how it looks like yet.

Attachments:

* [doc.core.fix.xhtml.errors.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/9d0/9d0ad0796b568715463fa10d7ae81c31cc407b0d/0e09bed1c47e69267779bf601294c5242f915433.patch) (28186 bytes) - added by techtonik on 2011-04-03 18:06:19Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5018 5018
type__enhancement enhancement
reporter__techtonik techtonik
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__core core
keywords__ 
time__1301853963000000 1301853963000000
changetime__1301861149000000 1301861149000000
version__None None
owner__ 
cc__techtonik
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

Thanks for your attention to the docs, techtonik.  I think we _don't_ want to make this kind of change right now though.  We're most of the way through converting all our long-form documentation from Lore XHTML to Sphinx reStructured Text.  Markup changes are more likely to confuse the conversion tool we're using, and these issues will all be fixed automatically once the switch is complete.

Thanks anyway, though.  If you'd like to follow the progress of the conversion, or help out with it, [#4500](https://github.com/twisted/twisted/issues/4500) is the ticket to check out.
